stable/osg-frontier-squid/osg-frontier-squid/Chart.yaml

```
@@ -5,4 +5,4 @@ appVersion: 4.4-2.1
 description: A Helm chart for configuration and deployment of the Open Science Grid's Frontier Squid application.
 name: osg-frontier-squid
 # Chart version
-version: 1.4.6
+version: 1.4.2
```

```suggestion
version: 1.4.2
```
Chart version *cannot* go down